src/main/resources/db/migration/V68__bruk_vedtaksperioder_i_beregningsgrunnlag_tilsyn_barn.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE vedtak
+SET data = replace(
+        replace(data::text, '"stønadsperioderGrunnlag"', '"vedtaksperiodeGrunnlag"'),
+        '"stønadsperiode"', '"vedtaksperiode"'
+           )::jsonb
+WHERE data ->> 'type' IN ('INNVILGELSE_TILSYN_BARN', 'OPPHØR_TILSYN_BARN');
```

Comment:
Eneste gotcha jeg kan tenke meg med dette scriptet er for opphør. Her har vi begrunnelse som er fritekst, og hvis saksbehandler har skrevet f.eks. 
```
{
  "begrunnelse": "Dette gjelder stønadsperiode for 2024",
  "stønadsperiode": { "fra": "2024-01-01", "til": "2024-12-31" }
}
```
vil det blir overskrevet med 
```
{
  "begrunnelse": "Dette gjelder vedtaksperiode for 2024",
  "vedtaksperiode": { "fra": "2024-01-01", "til": "2024-12-31" }
}
```
Dette er sannsynligvis ikke et problem, men verdt å tenkte på kanskje.

Reply:
Det går an å vurdere en alternativ løsning, der vi kopierer over verdien i JSON-feltene over til en ny nøkkel, og deretter sletter den gamle nøkkelen. Fordelen med dette er at vi aldri opererer på rein tekst. 

```
UPDATE vedtak
SET data = jsonb_set(
              jsonb_set(data, '{stønadsperioderGrunnlag}', data->'stønadsperioderGrunnlag', true) - 'stønadsperioderGrunnlag',
              '{stønadsperiode}', data->'stønadsperiode', true
          ) - 'stønadsperiode'
WHERE data ->> 'type' IN ('INNVILGELSE_TILSYN_BARN', 'OPPHØR_TILSYN_BARN');
```